public/lessons/db/mongodb.md

```diff
@@ -1,32 +1,10 @@
-# MongoDB Installation
 
-### For Windows users
+# Setup
+* Download [MongoDB Compass](https://www.mongodb.com/try/download/compass)
```

Comment:
We should keep this as the install for the community version. That comes with compass by default, but also allows us to connect to our database through other means, whereas I'm not sure we can do that if we just install compass.